.github/workflows/api-tests-with-private-packages.yml

```diff
@@ -11,7 +11,9 @@ defaults:
     working-directory: api
 
 env:
+  FLAGSMITH_SAML_REVISION: v1.6.1
   FLAGSMITH_RBAC_REVISION: v0.7.0
+  FLAGSMITH_AUTH_CONTROLLER_REVISION: v0.1.0
```

Comment:
Auth-controller version now has to be inferred from pyproject now that #4174 is merged.

Reply:
This is done